samples/subsys/video/capture_to_lvgl/src/main.c

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/video.h>
+#include <lvgl.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(main);
+
+#define VIDEO_DEV_SW "VIDEO_SW_GENERATOR"
+
+void draw_label(void)
+{
+	lv_obj_t *label = lv_label_create(lv_scr_act());
+
+	lv_label_set_text(label, "LVGL on Zephyr!");
+	lv_obj_align(label, LV_ALIGN_CENTER, 0, 30);
+
+	lv_task_handler();
+}
+
+void draw_lvgl_logo(void)
+{
+	LV_IMG_DECLARE(img_lvgl_logo);
+	lv_obj_t *img1 = lv_img_create(lv_scr_act());
+
+	lv_img_set_src(img1, &img_lvgl_logo);
+	lv_obj_align(img1, LV_ALIGN_CENTER, 0, -10);
+
+	lv_task_handler();
+}
+
+void display_splash_screen(void)
+{
+	draw_label();
+	draw_lvgl_logo();
```

Comment:
is this really needed? Unless I am missing something I don't see the added value of this image (but it certainly adds to the binary image size :))

In fact, I am really questioning the justification of having a splash screen in the first place, and I would jump straight into the actual video sample.

Reply:
now that I think about it some more, I realize that maybe your intention was to show "a bit more" of LVGL in your sample than just video stuff. In this case, what I think would be cool, and likely pretty simple, would be to show how easy it is to add the ability to drag the video (of course this requires a touch screen) https://docs.lvgl.io/master/examples.html#make-an-object-draggable ... or something similar?

Reply:
I agree to keep it simple. 

About the drag suggestion, can we leave this improvement for the future?

Reply:
The splash screen was removed.